### ISSUE TITLE
sitewide 508-defect-2 [COGNITION, SCREEN READER, MOBILE]: Mobile and screen reader users SHOULD know there is relevant info after the call-to-action button

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#16731]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I need to know that there is additional relevant information after the call-to-action. Since there is a link to Sign in or create an account, I may just select that button and never know that there is information that clarifies the form, process, next steps, and how to get help completing the form.

## Details

**This is an issue across the site.** 
[View the original issue here.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16182)

While this is a sitewide issue that should be addressed globally, the team should address this within this work. The Healthcare Questionnaire team also identified UX issues for sighted users with this pattern, especially on mobile.

The placement of the call-to-action (CTA) to sign in to use the service is at the top of the page. For sighted users on desktop it is clear there is additional information available. For screen reader users, there is no indication there is additional information that could be helpful. It is possible to add `aria-describedby` to the button to associate it with the following information. 

However, a larger issue is that this information is also not readily apparent for mobile users. It may be useful to consider removing the CTA from the top, ensuring users go through all the educational material before they use the CTA at the bottom of the page. This may require adopting a show/hide pattern for the subway steps, [as gov.uk does](https://www.gov.uk/apply-online-for-student-finance).

## Acceptance Criteria

- [ ] The Sign in or create an account button has an `aria-describedby` associated with a unique id on the relevant following information.
- [ ] Consider if it is desirable, viable, and feasible to remove the top CTA, potentially looking into research to see if this is desirable for users (esp mobile and screen reader), viable & feasible check with VSP (Trevor is aware).

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: this project is in staging; the issue exists in production
* Devices: mobile and screen readers especially impacted

## Steps to Recreate

1. Enter `<https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction>` in browser
2. Start screenreading device listed in Environment
3. Start to read the page, without looking at it
4. Verify that there is no indication of the following support material
5. Optional: enter the url above on a mobile device
6. Verify that the CTA takes up the entire screen, and there is no indication of the following support content

#### Current

```html
<button class="va-button-primary">Sign in or create an account</button>
```

#### Recommendation

```html
<button aria-describedby="unique_id" class="va-button-primary">Sign in or create an account</button>

<div id="unique_id"… <!-- this could be on a container div or the h2 -->
```

### Option 2 - explore removing the top CTA

This is something that should be discussed with VSP, cc @1Copenut @sshein. On VSA we can:

1. Discuss the problem found
2. Explore and document actions
3. Document a solution that works for VSA teams
4. Propose to VSP design system council for review

Note: I believe this issue is also being discussed on the Healthcare work, and it may be helpful to connect with Kristen McConnell and myself. It could be helpful is VSA teams had a collaboration session to review this flow and brainstorm potential fixes. For example, Have the landing page only be the "Sign in or create an account", and then once signed-in, provide the subway steps task list before they begin the process. Others may have better ideas.

## WCAG or Vendor Guidance (optional)

* [WCAG 2.1 Level A - 1.3.2 Meaningful Sequence](https://www.wuhcag.com/meaningful-sequence/) - Users who rely on assistive technology (such as a screen reader) to interpret content, require content to be presented in a meaningful order. If this is presented out of sequence, users may become disorientated and will not understand the content.

## Screenshots or Trace Logs

![Screen Shot 2020-11-13 at 5 15 18 PM](https://user-images.githubusercontent.com/57469/99128924-25128100-25da-11eb-994f-38d0f5aa1fb2.png)

**Comment by jason-gcio** [comment]
Adding `aria-describedby` to that button in the top section is not in our scope to adjust at this time.  Also, we would need more consultation about moving the CTA from the top of the page.

**Comment by joshkimux** [comment]
@jason-gcio this is a part of a larger conversation, so let's keep our eyes on this in the future 👍 
@1Copenut would like to get your thoughts on this too to see how we can get the conversation moving with the right people

**Comment by 1Copenut** [comment]
@joshkimux Please see my comment on 16182: https://github.com/department-of-veterans-affairs/va.gov-team/issues/16182#issuecomment-747092893

LMK if you'd like to discuss further in a video chat or DM.

**Comment by joshkimux** [comment]
@1Copenut saw it and acknowledged. Thank you for weighing in!
